### ISSUE TITLE
replace existing icons with updated version on trello

### ISSUE THREAD

**Issue by piggyslasher** [lavab/web#375]
contacts,
add contact

@vLooz - how about these add/cancel/delete/edit icons - should i replace the existing ones?

**Comment by vLooz** [comment]
Not necessarily @piggyslasher. Some logos are for situations, where we did not have logos so far.

REPLACE

For "contacts", "add new contact" and "edit", the old logos should be replaced with the new ones. See here: https://github.com/lavab/web/issues/303 & https://github.com/lavab/web/issues/325

MISSING LOGOS

"Put Back" is a new Logo to move mails, which were deleted or marked as spam, back to either Inbox or the folder, they have been moved from. See here: https://github.com/lavab/web/issues/330

The circular "add" and "delete" logos are only for adding and deleting emails in a contact, see here: https://github.com/lavab/web/issues/325

Just look at the styleguides, where which Icon should be placed. When something is not logically, inconsistent or confusing, let me know, maybe I've messed up :-)
